src/pages/amazon-sales-channel/best-practices/logging-troubleshooting.md

```diff
@@ -0,0 +1,253 @@
+
+```bash
+wskdebug --inspect-brk=11932 <action_name> <path_action_name.js> -l -v
+```
+
+* `@adobe/wskdebug 1.1.3` &#9989;
+* `@openwhisk/wskdebug 1.2.0` &#9989;
+* `@openwhisk/wskdebug 1.3.0` &#10060;
+* `@openwhisk/wskdebug 1.4.0` &#10060;
+
+The error return from the last two versions is:
+
+`FetchError: request to http://0.0.0.0:53362/init failed, reason: connect ECONNREFUSED 0.0.0.0:53362`
+
+<InlineAlert variant="info" slots="text" />
+
+If you encounter any issues, check the `openwhisk` local log file (`openwhisk-local.log.txt`), which should be inside the `dist` folder in your project.
+
+### Limitations and known issues
+
+* You can not use `wskdebug` to debug your actions if you are using `aio-lib-state` or `aio-lib-files`. See [common issues](https://developer.adobe.com/app-builder/docs/getting_started/common_troubleshooting/#debugging-errors-with-state-and-files-sdk)
+
+* Breakpoints might not pausie when debugging an action.  [Troubleshoot Asset Compute extensibility](https://experienceleague.adobe.com/docs/experience-manager-learn/cloud-service/asset-compute/troubleshooting.html?lang=en#debug) provides more information.
+
+* The `aio app run --local` command does not work with Nodejs18, Currently, only Nodejs16 is supported.
+
+* You cannot run your application with the `--local flag` in any of the following cases:
+
+  * The limits/timeout configured in your `app.config.yml` file are higher than 300000 ms. Setting the value too high causes the following error:
+  
+    ```text
+    ›   Error: PUT http://localhost:3233/api/v1/namespaces/guest/actions/amazon-app/scheduled-amazon-sync-products Returned HTTP 400 (Bad Request) --> "The request content was malformed:
+    ›   requirement failed: duration 400000 milliseconds exceeds allowed threshold of 300000 milliseconds"
+    ```
+
+  * You have a dependency on the `lib-state` storage library, which causes the following error:
+
+    ```text
+    "code": "ERROR_BAD_ARGUMENT",
+    "message": "[TvmLib:ERROR_BAD_ARGUMENT] ow.auth" is required"
+    "stacktrace": "TvmLibError: [TvmLib:ERROR_BAD_ARGUMENT] "ow.auth" is required
+    ```
+
+  * You have triggers (scheduled runtime actions) configured. This will show the following error:
+
+    ```text
+    Error: POST http://localhost:3233/api/v1/namespaces/whisk.system/actions/alarms/interval Returned HTTP 403 (Forbidden) --> "The supplied authentication is not authorized to access 'whisk.system/alarms'."
+    ```
+
+### Timeout issues
+
+Based on the development team's experience developing Amazon Sales Channel, a 503 error may occur when an action exceeds its allotted time. However, this error message may not always provide sufficient information, especially when the action is invoked from a React UI. In such cases, it is recommended to investigate further using the activation ID to gain more insights. If the result obtained from activation ID indicates a timeout error, it will typically display a message such as:
```

Comment:
```suggestion
Based on the development team's experience developing Amazon Sales Channel app, a 503 error may occur when an action exceeds its allotted time. However, this error message may not always provide sufficient information, especially when the action is invoked from a React UI. In such cases, it is recommended to investigate further using the activation ID to gain more insights. If the result obtained from activation ID indicates a timeout error, it will typically display a message such as:
```